dev/load-test/test-recipes/csn-load-test/churn-driver.py

```
@@ -0,0 +1,336 @@
+                if cont:
+                    kwargs["_continue"] = cont
+                resp = self.api.list_namespaced_custom_object(**kwargs)
+                for c in resp.get("items", []):
+                    total += 1
+                    ann = c.get("metadata", {}).get("annotations") or {}
+                    if ann.get(ADOPTED_ANNOTATION):
+                        adopted += 1
+                cont = resp.get("metadata", {}).get("continue")
+                if not cont:
+                    break
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)